app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -22,26 +22,19 @@ input[type=text].gem-c-input {
 
   // Disable inner shadow and remove rounded corners
   appearance: none;
+
+  &.gem-c-input:focus {
+    outline: $gem-focus-width solid $gem-focus-colour;
+  }
+
+  &.gem-c-input--error {
+    border: $gem-border-width-error solid $gem-error-colour;
+  }
 }
 // scss-lint:enable QualifyingElement
 
 @media (min-width: 40.0625em) {
-  .govuk-c-input {
+  .gem-c-input {
```

Comment:
I wasn't sure whether to delete this or update it. cc @nickcolley 
I still need to check the specificity of it too: see https://github.com/alphagov/govuk_publishing_components/pull/241/files#diff-57a28dc54e1ce322564d3c1e15811c5fR7